lib/mailserver.js

```diff
@@ -16,8 +16,8 @@ const logger = require('./logger')
 const smtpHelpers = require('./helpers/smtp')
 const { calculateBcc } = require('./helpers/bcc')
 const outgoing = require('./outgoing')
-const TurndownService = require('turndown')
-const marked = require('marked')
+const createDOMPurify = require('dompurify')
+const { JSDOM } = require('jsdom')
```

Comment:
Post-merge and npm publish, I realized that `jsdom` is not explicitly included in the package.json dependencies so this fails on the latest version on npm. This can work locally as jsdom is included as a dev dep's own dependency, but this isn't included in prod.